src/overlays/actors/ovl_Obj_Etcetera/z_obj_etcetera.h

```diff
@@ -3,13 +3,32 @@
 
 #include "global.h"
 
+#define OBJETCETERA_TYPE(thisx) (((thisx)->params & 0xFF80) >> 7)
```

Comment:
```suggestion
#define DEKU_FLOWER_TYPE(thisx) (((thisx)->params & 0xFF80) >> 7)
```
We know what it is, so might as well call it this.